### PR TITLE
Experiment with different fonts

### LINKED ISSUE #508: Font and typography: give it another look once redesign and content is in place
This is a meta ticket to collect all the fixes needed regarding fonts and typography.

We need to give another look at the fonts and typography once redesign is done. The proposed design did not force us to use certain font (this was requested) but left it to us to decide. Few proposals were given:

- **Zilla Slab**
- **Krub**
- **Sora**
- **Open Sans**

### Collected feedback that we should consider:

- The typography on mobile is too splashy: too big and lack of security margins. Consequentially, it is harder to sync and parse the content (#600)
- Self host the font (#601, #551)

<details><summary>Original message of the issue</summary>
I'm not happy with the font choices for the redesign. They feel arbitrary and not specifically a good fit for our project.

- **Zilla Slab** looks like a typewriter, so not modern.
- **Krub** has many curves which don't fit the rest of the design with a lot of sharp edges
- **Sora** feels messy and unfocused with all the small details
- **Open Sans** is actually not bad. it's perfectly readable and for good reason the second most popular font on google fonts

Once we select fonts we should serve them from the nixos.org.

The demo uses **Sora** for the headings and **Krub** for the paragraphs in the current state.

Screenshot from 2020-08-28 21-46-00

I find the fonts too fancy and detailed. The round w and x distract me from the content. I have autism and dyslexia, so i'm very easy distracted and it's hard for me to read. So it is an accessibility issue.

Here are my alternative suggestions:

Screenshot from 2020-08-28 17-57-55

**Source Sans Pro + Open Sans**

Perfectly readable, heavy heading.

Screenshot from 2020-08-28 18-01-57

**Overpass + Open Sans**

(https://reproducible-builds.org/ uses Overpass for all text 😬)

i find the first suggestion more readable, the second looks sharper and might fit the nix snowflake and lambda better

We can ask the designer if these fonts fit together and to the rest of the new design. Source Sans Pro + Open Sans is a popular pairing according to google fonts.
</details>

### PR DESCRIPTION
**Pull request by garbas**
This PR tries to explore different font combinations. This was one of the items on the TODO list after the redesign.

There are 3 fonts we need to select:

1.) **Primary**: font used for general text
2.) **Secondary**: font used for titles
3.) **Monospace**: font used for code examples

If you wish to suggest new combination please write in the comments.

# Font combinations

- [Primary: Krub
   Secondary: Sora
   Monospace: Fira Mono](https://5fe9dcefa33d901b2ee214af--nixos-homepage.netlify.app/)
- [Primary: Open Sans
   Secondary: Zilla Slab
   Monospace: Fira Mono](https://5fe9de0782357550f26a3e08--nixos-homepage.netlify.app/)
- [Primary: Open Sans
   Secondary: Source Sans Pro
   Monospace: Fira Mono](https://5fe9ded393c2cc3f87f54c0c--nixos-homepage.netlify.app/)
- [Primary: Open Sans
   Secondary: Overpass
   Monospace: Fira Mono](https://5fe9df7832319c4dbeea6892--nixos-homepage.netlify.app/)
- [Primary: Roboto
   Secondary: Source Sans Pro
   Monospace: Fira Mono](https://5fe9e02232319c5186ea6043--nixos-homepage.netlify.app/)
- [Primary: Roboto
   Secondary: Overpass
   Monospace: Fira Mono](https://5fe9e08302f86e3b7b1ccb6d--nixos-homepage.netlify.app/)
- [Primary: Montserrat
   Secondary: Source Sans Pro
   Monospace: Fira Mono](https://5fe9e0d4d3fff9f9e2fec481--nixos-homepage.netlify.app/)
- [Primary: Roboto
   Secondary: Zilla Slab
   Monospace: Fira Mono](https://5fe9e0fcd3fff9fc80fec1fb--nixos-homepage.netlify.app/)
- [Primary: Meriweather
   Secondary: Lato
   Monospace: Fira Mono](https://5fe9e864bf1584e477d6edf2--nixos-homepage.netlify.app/)
- [Primary: Lato
   Secondary: Meriweather
   Monospace: Fira Mono](https://5fe9e7f3a3b5c8129f7dfaf7--nixos-homepage.netlify.app/)
- [Primary: Open Sans
   Secondary: Lato
   Monospace: Fira Mono](https://5fea573bf88342390af2db09--nixos-homepage.netlify.app/)

As always please keep the conversation civil especially when it comes to design. What looks good for somebody might not look good for somebody else. It is fine that you tell what you like and dislike, but it is even better if you tell why you dislike something (eg. not readable, ...). Thank you in advance.

fixes #508

### CODE REVIEW

**Comment by davidak** [comment]
Thanks @garbas!

Sora: hard to read because of the details and proportions
Krub: it bothers me that some characters are rounded where they shouldn't, like v, x and w. those unexpected details make it hard to read for me
Fira Mono: very nice font. the details like the . in 0 make it more readable instead of distracting
Zilla Slab: it feels a bit too heavy with those lines at top and bottom of characters
Open Sans: very neutral font. no unnecessary details. very readable
Source Sans Pro: very round font. readability is OK. might not be a good fit to the rest of design with lot of sharp edges
Overpass: readability OK. details fits design (sharp edges)
Roboto: like Open Sans very neutral and good readable. difference is hard to tell, even in direct comparison. Open Sans has more open c what i like more, but it's really personal taste
Montserrat: also very neutral and readable
Merriweather: looks like a classic typewriter font, better suited for printed text, not for a technical website. readability OK
Lato: very neutral, details don't steal attention from the content. very good readable in headings. in text, it's too thin to be good readable

tl;dr **my recommendation**

Primary: Open Sans
Secondary: Lato
Monospace: Fira Mono

i'm not sure if the difference between Primary and Secondary is big enough. so if we want to decide on this, i want to have a final look in a preview

there are also other combinations that would not bother me

**Comment by davidak** [comment]
i think that looks very good and consistent. you don't notice some special font, but just read the content

i would like to improve the font weight in a next step. the body seem to use the thin version which is harder to reed than regular. try how regular would look

Am 28. Dezember 2020 23:08:12 MEZ schrieb "github-actions[bot]" <notifications@github.com>:
>>
>🚀 Deployed on
>https://5fea573bf88342390af2db09--nixos-homepage.netlify.app>
>>
>-- >
>You are receiving this because your review was requested.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/NixOS/nixos-homepage/pull/663#issuecomment-751879117

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by nixinator** [comment]
https://www.fontspace.com/atari-font-f5956

or anything from

http://christophekohler.epizy.com/AmigaBlogWp/1991/11/15/step40-bitmap-font/

;-)

**Comment by hugolgst** [comment]
I think the first one is the best choice, readable and modern.

**Comment by garbas** [comment]
@hugolgst since the font combinations are not numbered can you copy paste the option that you selected. Also if you can expend a bit on why your choise is readable and modern?

**Comment by hugolgst** [comment]
This combination:
Primary: Krub
Secondary: Sora
Monospace: Fira Mono

I like it because the titles do not use serif fonts and I find it a bit easier to read.
Sans-serif titles are also modern.

**Comment by aherrmann** [comment]
Just noticed a small issue: when the selection produces an empty set then the side bar with the facet selectors vanishes and the user is stuck on an empty page without the ability to go back.

**Comment by turboMaCk** [comment]
My favorite is:

[Primary: Roboto
Secondary: Overpass
Monospace: Fira Mono](https://5fe9e08302f86e3b7b1ccb6d--nixos-homepage.netlify.app/)

**Comment by garbas** [comment]
I'm also in favor of 

Primary: Roboto
Secondary: Overpass
Monospace: Fira Mono

option.

I'm going to go ahead and implement it, since this has been open for some now and it would be nice to finalize this.

**Comment by davidak** [comment]
I'm also fine with it.

**Comment by samueldr** [comment]
RFC process SVG file fonts weren't updated.

image

Sadly, dealing with SVG is annoying for that :). Though a quick search/replace should do it.

**Comment by garbas** [comment]
@samueldr fixed

**Review by samueldr — APPROVED** [review]
